lib/cogctl/actions/bundles/create.ex

```diff
@@ -41,22 +47,74 @@ defmodule Cogctl.Actions.Bundles.Create do
   end
 
   defp do_create(endpoint, params) do
-    results = with {:ok, config}         <- Spanner.Config.Parser.read_from_file(params.file),
-                   {:ok, templates}      <- build_template_map(params.templates),
-                   {:ok, amended_config} <- maybe_add_templates(templates, config),
-                   {:ok, fixed_config}   <- Spanner.Config.validate(amended_config),
-                 do: Client.bundle_create(endpoint, fixed_config)
+    result = with {:ok, config} <- parse_config(params),
+                  {:ok, bundle} <- create_bundle(endpoint, params, config),
+                  do: {:ok, bundle}
 
-    case results do
+    case result do
       {:ok, bundle} ->
-        display_output("Created '#{bundle.name}' bundle")
+        display_output("Created #{bundle.name} bundle")
+
+        assign_to_relay_groups(endpoint, bundle, params)
+
+        status = Bundles.enabled_to_status(bundle.enabled)
+        bundle = Map.merge(bundle, %{status: status})
+
+        bundle_attrs = for {title, attr} <- [{"ID", :id}, {"Name", :name}, {"Status", :status}, {"Installed", :inserted_at}] do
+          [title, Map.get(bundle, attr) |> to_string]
+        end
+
+        commands = for command <- bundle.commands do
+          [command["name"], command["id"]]
+        end
+
+        display_output("""
+        #{Table.format(bundle_attrs, false)}
+
+        Commands
+        #{Table.format([["NAME", "ID"]|commands], true)}
+        """ |> String.rstrip)
       {:error, messages} when is_list(messages) ->
         # Map over messages and convert any validation errors
         # into strings so cogctl can display them
         Enum.map(messages, &format_validation_reason/1) |> display_error
     end
   end
 
+  defp parse_config(params) do
+    with {:ok, config}         <- Spanner.Config.Parser.read_from_file(params.file),
+         {:ok, templates}      <- build_template_map(params.templates),
+         {:ok, amended_config} <- maybe_add_templates(templates, config),
+         {:ok, fixed_config}   <- Spanner.Config.validate(amended_config),
+         do: {:ok, fixed_config}
+  end
+
+  defp create_bundle(endpoint, params, config) do
+    params = Map.put(params, "config", config)
+    Client.bundle_create(endpoint, params)
+  end
+
+  defp assign_to_relay_groups(endpoint, bundle, %{"relay-groups": relay_group_names}) do
+    relay_groups = String.split(relay_group_names, ",")
+
+    Enum.reduce_while(relay_groups, {:ok, []}, fn relay_group, {:ok, acc} ->
+      result = Client.relay_group_add_bundle(%{name: relay_group}, %{bundle: bundle.name}, endpoint)
```

Comment:
This function changed a bit with the updates to cog_api. Now it takes a list of bundles and add them all to the relay group.

`Client.relay_group_add_bundles(%{name: relay_group}, %{bundles: [bundle.name]}, endpoint)`